examples/models/llama2/export_llama_lib.py

```diff
@@ -563,6 +563,22 @@ def _export_llama(modelname, args) -> LLMEdgeManager:  # noqa: C901
                 shares=args.num_sharding,
             )
 
+        from functools import partial
+
+        from executorch.backends.qualcomm.quantizer.custom_annotation import (
+            get_custom_quant_ios_dtype,
+        )
+
+        tag_quant_io(
```

Comment:
Can it be part of [`_transform()` function](https://github.com/pytorch/executorch/blob/main/backends/qualcomm/utils/utils.py#L292-L294)?

Reply:
I think if sharding is enabled, this function should be executed after model sharding.
Because it will tag the tensors between sharding.